spec/features/dams_unit_spec.rb

```diff
@@ -92,5 +97,11 @@
     
     # does not find this collection because it belongs to other unit
     expect(page).to_not have_selector('a', :text => 'Test Collection')
-  end  
+  end
+  
+  scenario 'rdcpt collection search' do
+    visit dams_unit_collections_path('rdcpt')
```

Comment:
The overall space change seems 👍 

While the code for this test unit is indeed `rdcpt`, is that a description of the scenario being tested here? It seems to be about testing the contextual search display text. "Search and browse contextually within a unit", perhaps, or something like that. Curious what others think.